packages/contracts/test/deploy/updateTo1_3_0.ts

```diff
@@ -1,16 +1,29 @@
 import {expect} from 'chai';
 
 import {deployments} from 'hardhat';
-import {initForkAndFixture} from '../test-utils/fixture';
-import {v1_0_0_active_contracts} from '@aragon/osx-versions';
+import {
+  ForkOsxVersion,
+  initForkForOsxVersion,
+  initializeDeploymentFixture,
+} from '../test-utils/fixture';
+import {activeContractsList as v1_2_0_activeContracts} from '@aragon/osx-ethers-v1.2.0';
```

Comment:
Do we need the test against 1.2.0 or is this a leftover?

Reply:
contracts-ethers have the active contracts of mumbai and goerli as well